config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
Copying boot.img is not a must, but anyway we do need the configuration file, so the folder is needed.

Not sure about using snapd database, it might be an option, which might generic for different bootloaders. In the case of android-boot using it or not depends on whether tools would need to be added to the initramfs or not.